.pre-commit-config.yaml

```diff
@@ -0,0 +1,31 @@
+# See https://pre-commit.com/hooks.html for info on hooks
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.4.0
+    hooks:
+      - id: check-added-large-files
+      - id: check-ast
+      - id: check-case-conflict
+      - id: check-toml
+      - id: check-yaml
+      - id: debug-statements
+      - id: detect-private-key
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+
+  - repo: https://github.com/psf/black
+    rev: 19.10b0
+    hooks:
+      - id: black
+
+  - repo: https://gitlab.com/pycqa/flake8
+    rev: 3.8.4
+    hooks:
+      - id: flake8
+        args: [--max-line-length=88]
+        language_version: python3.7
```

Comment:
Personally I'm running openSUSE Tumbleweed and there are only Packages for Python 3.6 and 3.8 in the default Repos. So I either have to adjust the language_version in my local copy or run into issues in the pre-commit hooks. I was thinking if there is a way to define a minimum Python version instead a fixed one? I played around with argument "--min-python-version" as described [here](https://stackoverflow.com/a/60680277) but without success yet.

Reply:
Oh, that's good to know. Actually, I don't even remember why that line was originally added. I imagine there was a reason at the time to override the Flake8 hook's default Python `language_version`, but since I don't recall, I have simply removed this line.